### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by dahlbyk**
Eleven years after 3033fa53c2117b0d35679461978cc0bcfd6a8ad6...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ Table of contents:
 posh-git is a PowerShell module that integrates Git and PowerShell by providing Git status summary information that
 can be displayed in the PowerShell prompt, e.g.:
 
-![C:\Users\Keith\GitHub\posh-git [master ≡ +0 ~1 -0 | +0 ~1 -0 !]> ][prompt-def-long]
+![C:\Users\Keith\GitHub\posh-git [main ≡ +0 ~1 -0 | +0 ~1 -0 !]> ][prompt-def-long]
```
Comment (rkeithhill) [inline README.md:23]: 👍

File: README.md [hunk 2]
```diff
@@ -509,22 +511,22 @@ function prompt {
...
-[prompt-path]:     https://github.com/dahlbyk/posh-git/wiki/images/PromptOrangePath.png    "~\GitHub\posh-git [master ≡]> "
-[prompt-swap]:     https://github.com/dahlbyk/posh-git/wiki/images/PromptStatusFirst.png   "[master ≡] ~\GitHub\posh-git> "
-[prompt-two-line]: https://github.com/dahlbyk/posh-git/wiki/images/PromptTwoLine.png       "~\GitHub\posh-git [master ≡]&#10;> "
-[prompt-custom]:   https://github.com/dahlbyk/posh-git/wiki/images/PromptCustom.png        "[master ≡] ~\GitHub\posh-git&#10;02-18 14:04:35 38> "
-[prompt-custom-wpathdelim]:   https://github.com/dahlbyk/posh-git/wiki/images/PromptCustomDelim.png        "[master ≡] {~\GitHub\posh-git}&#10;02-18 14:04:35 38> "
-[prompt-error1]:   https://github.com/dahlbyk/posh-git/wiki/images/PromptError1.png        "~\GitHub\posh-git [master ≡]&#10;! 07-01 22:36:31> "
-[prompt-error2]:   https://github.com/dahlbyk/posh-git/wiki/images/PromptError2.png        "~\GitHub\posh-git [master ≡]&#10;(1) 07-01 22:32:28> "
+[prompt-def-long]: https://github.com/dahlbyk/posh-git/wiki/images/PromptDefaultLong.png   "~\GitHub\posh-git [main ≡ +0 ~1 -0 | +0 ~1 -0 !]> "
```
Comment (rkeithhill) [inline README.md:519]: Hmm, I suppose we should update the wiki images at some point ... later.  :-)

File: src/posh-git.psd1 [hunk 3]
```diff
@@ -4,7 +4,7 @@
 RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.0.0.0'
+ModuleVersion = '1.0.0.1'
```
Comment (rkeithhill) [inline src/posh-git.psd1:7]: I take it you already have a built version of 1.0.0 set aside to publish to the gallery?

**Review by rkeithhill — APPROVED** [review]

File: src/posh-git.psd1 (hunk 3 above)
Comment (dahlbyk) [reply src/posh-git.psd1:7]: > I take it you already have a built version of 1.0.0 set aside to publish to the gallery?

image

Comment (rkeithhill) [reply src/posh-git.psd1:7]: The 3rd and 4th downloads are mine. :-)

**Pull request merged**